### PR TITLE
fix(telephony.alias): fix call forwarding option

### PR DESCRIPTION
**Pull request by JeremyDec**
MANAGER-2079

(Linked to : https://github.com/ovh-ux/ng-ovh-telecom-universe-components/pull/75)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "@ovh-ux/ng-ovh-http": "^4.0.1-beta.0",
     "@ovh-ux/ng-ovh-sso-auth": "^4.0.0",
     "@ovh-ux/ng-ovh-sso-auth-modal-plugin": "^3.0.0",
-    "@ovh-ux/ng-ovh-telecom-universe-components": "^2.0.2",
+    "@ovh-ux/ng-ovh-telecom-universe-components": "^2.0.x",
```
Comment (FredericEspiau) [inline package.json]:
```suggestion
    "@ovh-ux/ng-ovh-telecom-universe-components": "^2.0.0",
```
is enough :)

Comment (antleblanc) [reply package.json]: Otherwise you can use `yarn upgrade-interactive --latest` and pick the latest release.

Comment (FredericEspiau) [reply package.json]: He told me it didn't work 🤔

Comment (JeremyDec) [reply package.json]: A simple `yarn upgrade` didn't work :) I didn't used `-interactive -latest` flag

Comment (antleblanc) [reply package.json]: @JeremyDec which version of `yarn` are you using?

Comment (JeremyDec) [reply package.json]: @antleblanc `yarn version` : `yarn version v1.13.0`

**Pull request merged**